python/yugabyte_db_thirdparty/builder.py

```diff
@@ -592,6 +604,12 @@ def init_linux_clang7_flags(self, dep: Dependency) -> None:
             self.compiler_flags.append('--gcc-toolchain={}'.format(
                 self.compiler_choice.get_linuxbrew_dir()))
 
+        if self.toolchain and self.toolchain.toolchain_type == 'llvm7':
+            # This is needed when building with Clang 7 but without Linuxbrew.
+            # TODO: this might only be needed due to using an old version of libunwind that is
+            # different from libunwind included in the LLVM 7 repository. Just a hypothesis.
+            self.ld_flags.append('-lgcc_s')
+
```

Comment:
Could we create a separate function for this workaround and use it here and in i4cu bulid?

Reply:
Actually, this is probably not needed here. The flag in builder.py will apply to all dependencies.